lbz/dev/test.py

```diff
@@ -93,3 +93,4 @@ def _process(
                 headers=headers,
             )
         )()
```

Comment:
Shouldn't you use `.react()` here instead? You marked `__call__()` as deprecated 😉